addons/fleet/wizard/fleet_vehicle_send_mail.py

```diff
@@ -0,0 +1,70 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models, _
+
+
+class FleetVehicleSendMail(models.TransientModel):
+    _name = 'fleet.vehicle.send.mail'
+    _inherit = 'mail.composer.mixin'
+    _description = 'Send mails to Drivers'
+
+    vehicle_ids = fields.Many2many('fleet.vehicle', string='Vehicles', required=True)
+    author_id = fields.Many2one('res.partner', 'Author', required=True, default=lambda self: self.env.user.partner_id.id)
+    template_id = fields.Many2one(domain=lambda self: [('model_id', '=', self.env['ir.model']._get('fleet.vehicle').id)])
+    attachment_ids = fields.Many2many(
+        'ir.attachment', 'fleet_vehicle_mail_compose_message_ir_attachments_rel',
+        'wizard_id', 'attachment_id', string='Attachments')
+
+    @api.onchange('template_id')
+    def _onchange_template_id(self):
+        self.attachment_ids = self.template_id.attachment_ids
```

Comment:
maybe attachment_ids could be a related field on `template_id.attachment_ids`

Reply:
Related fields are readonly by default, so it wouldn't change the original template AFAIK

Reply:
but then we can't add or remove attachments to our mail :D

Reply:
ah :facepalm: